core/class-plugin-installer.php

```diff
@@ -96,7 +99,7 @@ public function install_plugin( $plugin_zip = '' ) {
 			}
 			// clear the cache so we're using fresh data.
 			wp_cache_flush();
-			$upgrader       = new \Plugin_Upgrader();
+			$upgrader       = new \Plugin_Upgrader;
```

Comment:
Let's keep the brackets here.
```suggestion
			$upgrader       = new \Plugin_Upgrader();
```

Reply:
Ok iv disabled warnings for this